Dockerfile

```diff
@@ -8,11 +8,20 @@ RUN composer install \
     --no-scripts \
     --prefer-dist
 
+FROM python:3 AS parser
```

Comment:
@wurstbrot I wrote this merge script in python because it's the language I know. Feel free to rewrite it in php. Consider that it's just used at build time.